samples/bluetooth/central_dfu_smp/src/main.c

```diff
@@ -303,6 +303,7 @@ static void smp_echo_rsp_proc(struct bt_dfu_smp *dfu_smp)
 		}
 
 		cbor_error = cbor_value_to_pretty_advance(stdout, &value);
+		fflush(stdout);
```

Comment:
Is the  `printk("\n")` below maybe meant to be a newline to stdout instead? Then when the stream was line buffered it would have been flushed by that newline?